### PR TITLE
Fonts and png fix

### PR DESCRIPTION
**Pull request by TeodorKolev**
Fonts and png fix

### CODE REVIEW

**Comment by alicoding** [comment]
Do you mind removing the rest of the loader that related to your loader? it seems like you have merged them all into one.

**Comment by TeodorKolev** [comment]
Please define rest of the loader

File: webpack.loaders.js [hunk 1]
```diff
@@ -47,5 +47,8 @@ module.exports = [
 		test: /\.png/,
 		exclude: /(node_modules|bower_components)/,
 		loader: "url-loader?limit=10000&mimetype=image/png"
+	},
+	{ test: /\.(png|woff|woff2|eot|ttf|svg)$/,
```
Comment (alicoding) [inline webpack.loaders.js]: Look at line `17`-`47` those are duplicate of yours?

Comment (TeodorKolev) [inline webpack.loaders.js]: Yes but when I install font without that code it did not working

Comment (alicoding) [inline webpack.loaders.js]: So if you keep yours and removing the rest will that work?

Comment (TeodorKolev) [inline webpack.loaders.js]: I will try

Comment (TeodorKolev) [inline webpack.loaders.js]: Yes It is working only with my code. It is not working without it.

Comment (alicoding) [inline webpack.loaders.js]: So, can you remove the rest of the loader from 17-47 and just keep yours?

Comment (TeodorKolev) [inline webpack.loaders.js]: Yes, I can

Comment (TeodorKolev) [inline webpack.loaders.js]: why did you merged?

Comment (alicoding) [inline webpack.loaders.js]: not really...

**Pull request merged**